lib/src/nodes/config_node_include.cc

```diff
@@ -0,0 +1,40 @@
+#include <internal/nodes/config_node_include.hpp>
+#include <internal/nodes/config_node_simple_value.hpp>
+
+using namespace std;
+
+namespace hocon {
+
+    config_node_include::config_node_include(vector<unique_ptr<abstract_config_node>> children,
+                                             config_include_kind kind) :
+        _children(move(children)), _kind(kind) { }
+
+    token_list config_node_include::get_tokens() const {
+        token_list tokens;
+        for (auto&& node : _children) {
+            for (auto&& token : node->get_tokens()) {
+                tokens.push_back(token);
+            }
+        }
+        return tokens;
+    }
+
+    vector<unique_ptr<abstract_config_node>> const& config_node_include::children() const {
+        return _children;
+    }
+
+    config_include_kind config_node_include::kind() const {
+        return _kind;
+    }
+
+    string config_node_include::name() const {
+        for (auto&& node : _children) {
+            config_node_simple_value* simple = dynamic_cast<config_node_simple_value*>(node.get());
```

Comment:
It's not a common pattern, but you can declare a variable and check if it's null in the expression of an if block. So
```
if (config_node_simple_value* simple = dynamic_cast<config_node_simple_value*>(node.get())) {
    return simple->get_value()->transform_to_string();
}
```
should work.